src/main/java/com/smalaca/messagesender/exceptions/inmemory/MessageDoesNotExistException.java

```diff
@@ -3,7 +3,7 @@
 public class MessageDoesNotExistException extends RuntimeException {
 
     public MessageDoesNotExistException(String messageId) {
-        super("Message with id: " + messageId + " does not exists.");
+        super(messageId);
```

Comment:
szczerze mówiąc, to budowanie tej wiadomości mogło zostać w exceptionie. IMHO byłoby spójniej.